### PR TITLE
Fix Stochastic Gradient Descent Example

### PR DESCRIPTION
**Pull request by tylerjthomas9**
The example that is currently in the docs does not run. dtype, penalty, lrate, loss are not defined. This new version sets the default values for the parameters of cumlSGD, and copies Mini Batch SGD Regression's dtype for `pred_data['col1']`, `pred_data['col2']`. When running this example, I also got slightly different values for the output, so these were also updated. 

Old example:
```
import numpy as np
import cudf
from cuml.solvers import SGD as cumlSGD
X = cudf.DataFrame()
X['col1'] = np.array([1,1,2,2], dtype = np.float32)
X['col2'] = np.array([1,2,2,3], dtype = np.float32)
y = cudf.Series(np.array([1, 1, 2, 2], dtype=np.float32))
pred_data = cudf.DataFrame()
pred_data['col1'] = np.asarray([3, 2], dtype=dtype)
pred_data['col2'] = np.asarray([5, 5], dtype=dtype)
cu_sgd = cumlSGD(learning_rate=lrate, eta0=0.005, epochs=2000,
                fit_intercept=True, batch_size=2,
                tol=0.0, penalty=penalty, loss=loss)
cu_sgd.fit(X, y)
cu_pred = cu_sgd.predict(pred_data).to_array()
print(" cuML intercept : ", cu_sgd.intercept_)
print(" cuML coef : ", cu_sgd.coef_)
print("cuML predictions : ", cu_pred)
```

New example:
```
import numpy as np
import cudf
from cuml.solvers import SGD as cumlSGD
X = cudf.DataFrame()
X['col1'] = np.array([1,1,2,2], dtype = np.float32)
X['col2'] = np.array([1,2,2,3], dtype = np.float32)
y = cudf.Series(np.array([1, 1, 2, 2], dtype=np.float32))
pred_data = cudf.DataFrame()
pred_data['col1'] = np.asarray([3, 2], dtype=np.float32)
pred_data['col2'] = np.asarray([5, 5], dtype=np.float32)
cu_sgd = cumlSGD(learning_rate='constant', eta0=0.005, epochs=2000,
                fit_intercept=True, batch_size=2,
                tol=0.0, penalty='none', loss='squared_loss')
cu_sgd.fit(X, y)
cu_pred = cu_sgd.predict(pred_data).to_array()
print(" cuML intercept : ", cu_sgd.intercept_)
print(" cuML coef : ", cu_sgd.coef_)
print("cuML predictions : ", cu_pred)
```

### CODE REVIEW

**Comment by ajschmidt8** [comment]
@tylerjthomas9, thank you for bringing this to our attention! The HTML file that you've edited is automatically generated, so your changes would be overwritten if we merged it. Instead, the source code will need to be updated in [rapidsai/cuml](https://github.com/rapidsai/cuml). Since I'm just a docs maintainer, I'll defer to our cuML devs @dantegd and @cjnolet for more on this issue.

**Comment by dantegd** [comment]
Thanks for the PR @tylerjthomas9 ! Indeed AJ is correct, that file gets generated from our docstrings. So the place where the new example would need to go is in the cuML repo, specifically here: https://github.com/rapidsai/cuml/blob/branch-0.17/python/cuml/solvers/sgd.pyx#L136

**Comment by ajschmidt8** [comment]
Thanks, Dante. I will close this PR, but feel free to continue the conversation if necessary.
